submissions/LGeorge007/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,522 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Кинотеатр</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Choose your seats</h1>
+        <div class="ticket-legends">
```

Comment:
Fix styles for mobile

![image](https://user-images.githubusercontent.com/16196199/107275760-d64e7900-6a5a-11eb-805c-99935ebe0f08.png)